examples/guest_resource/.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Since everyone uses different IDEs, I suggest removing `vscode` files or adding them to the `.gitignore`.

Reply:
Agreed. I'm on neovim personally, so it doesn't even apply to me!

Again, it's a `cargo-component` template inclusion, which is what I've been using to build new components.

I'll remove it